From b8cfcfa8e3b7694102462b6d3d59ee7919897a00 Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Tue, 1 Dec 2020 10:19:00 +1100
Subject: [PATCH] winex11.drv: Move header order

The HAVE_X11_EXTENSIONS_XINPUT2_H needs to be moved into each of the files
to avoid a compile error with Status being undefined.
---
 dlls/winex11.drv/window.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dlls/winex11.drv/window.c b/dlls/winex11.drv/window.c
index 1e9d63ae808..22a330b3f88 100644
--- a/dlls/winex11.drv/window.c
+++ b/dlls/winex11.drv/window.c
@@ -37,6 +37,8 @@
 #include <X11/extensions/shape.h>
 #endif /* HAVE_LIBXSHAPE */
 
+#include "x11drv.h"
+
 /* avoid conflict with field names in included win32 headers */
 #undef Status
 #include "windef.h"
@@ -45,7 +47,7 @@
 #include "winuser.h"
 #include "wine/unicode.h"
 
-#include "x11drv.h"
+
 #include "wine/debug.h"
 #include "wine/server.h"
 #include "mwm.h"
-- 
2.29.2

